powerline/segments/common/bat.py

```diff
@@ -116,9 +116,14 @@ def _get_capacity(pl):
 
 					return _get_capacity
 				pl.debug('Not using win32com.client as no batteries were found')
-
-		from ctypes import Structure, c_byte, c_ulong, windll, byref
-
+		if sys.platform == 'cygwin':
+			pl.debug('Using cdll to communicate with kernel32 (Cygwin)')
+			from ctypes import Structure, c_byte, c_ulong, cdll, byref
```

Comment:
I would suggest moving anything, but cdll/windll import out of both branches because otherwise it looks like if you were using two identical lines. At least windll/cdll must be the first one because in this position it is easier to find differences.